packages/core-utils/src/ui.ts

```diff
@@ -1,19 +1,9 @@
-import bowser from "bowser";
-
 export function isMobile(): boolean {
   // TODO: consider using 3rd-party library?
   return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
     navigator.userAgent
   );
 }
-
-/**
- * Returns true if the user is using a [redacted] browser
- */
-export function isIE(): boolean {
```

Comment:
Although this is a breaking change, we can market that IE is no longer supported as of core-utils 11.0.0.